opentech/apply/funds/views.py

```diff
@@ -280,20 +279,23 @@ def get_form_kwargs(self):
         kwargs['request'] = self.request
         return kwargs
 
+    def form_invalid(self, form):
+        messages.error(
+            self.request,
+            mark_safe(
+                "There are errors with the Reviewers Form:" + form.errors.as_ul()
```

Comment:
I think you might want
`"There are errors with the Reviewers Form:" + form.errors.get('__all__').as_ul()`

Because the error in the message is adding in __all__, see this screenshot:
![image](https://user-images.githubusercontent.com/11393311/52563770-58562980-2dd0-11e9-95b0-a97a12db5200.png)

Reply:
@chris-lawton FYI whats the best way to style this?

we are returning a `<ul>` which sits along side the `<p>` and is causing an issue as the flex isnt assigned to it.

@emullaney If there are other errors it would be nice to render them nicely as well not just for the `__all__` case

Reply:
Can we wrap the `<p>` and the `<ul>` in a `<div>` to keep the flex alignment? So:
```
<div class="messages__inner">
        <svg class="messages__icon"><use xlink:href="#exclamation-point"></use></svg>
        <div>
            <ul>...</ul>    
            <p class="messages__copy">{{ message }}</p>
        </div>
        {% if close %}
            <a href="#" class="messages__button js-close-message">Ok</a>
        {% endif %}
    </div>
```

![image](https://user-images.githubusercontent.com/31627284/52574387-1949c780-2e14-11e9-877a-9236440246a6.png)